doc/api/tls.md

```diff
@@ -545,18 +545,19 @@ called:
 * `tlsSocket` {tls.TLSSocket} The `tls.TLSSocket` instance from which the
   error originated.
 
-### `server.addContext(hostname, context)`
+### `server.addContext(servername, context)`
```

Comment:
I think `hostname` is better. This is how TLS defines a "server name":

```
struct {
  NameType name_type;
  select (name_type) {
    case host_name: HostName;
  } name;
} ServerName;
```

It is correct that TLS does not allow IP addresses in the `HostName` field, but that is not necessarily true for `ServerName`. TLS simply doesn't support other `NameType`s so far, but it might do so in the future.

Reply:
I see the point, but... the extension has never changed and "server name" === "host name server name" in everyone's minds. I personally don't think it's worth it to expose this detail to users, nor have I seen any other API do it.

So I'm personally +1 for naming the parameter `servername` and stating things like "Setting the TLS ServerName to an IP address is not permitted", as we currently do.